packages/create-sitecore-jss/src/templates/react/data/dictionary/da-DK.yml

```diff
@@ -0,0 +1,4 @@
+styleguide-sample: Dette er en ordbogspost på dansk som en demonstration
```

Comment:
Check other initializers for handling of this file. Should be {{language}}.yml with generic 'translations'